CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ## New features
 - TBD - [#000](https://github.com/jertel/elastalert2/pull/000) - @some_elastic_contributor_tbd
+- HELM CHART - [#827] - exposes prometheus metrics to kubernetes pod service discovery mechanism @PedroMSantosD
```

Comment:
This should move down into the 4.x.x section. This top section is a template only. Also make sure the formatting matches the existing pattern. You can use `[Kubernetes]` instead of the `HELM CHART` prefix, for consistency with other changelog entries.

Reply:
moved to 2.x.x section